src/collatzConjecture.js

```diff
@@ -0,0 +1,17 @@
+export default function collatzConjecture(n){
+  if(n > 0){
```

Comment:
Check spacing on this file since you have changed it in the `setIntersection` file.